### PR TITLE
Adding babel to the webpack build

### PR DESCRIPTION
**Pull request by plumpNation**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -10,7 +10,13 @@
         "test": "karma start"
     },
     "dependencies": {
+        "@babel/core": "^7.0.0-beta.40",
+        "@babel/plugin-proposal-object-rest-spread": "^7.0.0-beta.40",
+        "@babel/preset-env": "^7.0.0-beta.40",
         "axe-core": "^2.6.1",
+        "babel-core": "^6.26.0",
```
Comment (plumpNation) [inline package.json]: Why are these here?

Comment (plumpNation) [reply package.json]: I was referring to the older babel-core and babel-plugin-transform-object-rest-spread deps.

**Review by alex-irt — APPROVED** [review]
